packages/okta-react/README.md

```diff
@@ -228,8 +219,17 @@ class App extends Component {
 }
 ```
 
+#### Alternate configuration using `Auth` object
```

Comment:
+1 on this section - very clear, very helpful

Reply:
The dev blog supports jsx as a language type. Maybe the new docs site does too?